packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect, useState } from 'react';
+import { Text } from '@deriv/components';
+import { Notifications as Announcement } from '@deriv-com/ui';
+import { StandaloneBullhornRegularIcon } from '@deriv/quill-icons';
+import { useHistory } from 'react-router-dom';
+import clsx from 'clsx';
+import { localize } from '@deriv/translations';
+import AnnouncementDialog from './announcement-dialog';
+import { BOT_ANNOUNCEMENTS_LIST, TAnnouncement, TNotifications } from './config';
+import './announcements.scss';
+import { MessageAnnounce, TitleAnnounce } from './announcement-components';
+import { getButtonAction } from './utils/accumulator-helper-functions';
+
+type TAnnouncements = {
+    is_mobile?: boolean;
+    handleTabChange: (item: number) => void;
+};
+
+const Announcements = ({ is_mobile, handleTabChange }: TAnnouncements) => {
+    const [is_announce_dialog_open, setIsAnnounceDialogOpen] = useState(false);
+    const [is_open_announce_list, setIsOpenAnnounceList] = React.useState(false);
+    const [selected_announcement, setSelectedAnnouncement] = React.useState<TAnnouncement | null>(null);
+    const [stored_notifications, setStoredNotifications] = React.useState({} as Record<string, boolean>);
+    const history = useHistory();
+    const [notifications, setNotifications] = useState([] as TNotifications[]);
+
+    const storeDataInLocalStorage = (temp_data: Record<string, boolean>) => {
+        localStorage?.setItem('bot-announcements', JSON.stringify(temp_data));
```

Comment:
For sure in the future, we could improve it. Since we wanted to deliver announcements(new important info) to the users ASAP we have done it on the frontend side. Later we can improve it and involve BE developers.